apps/dapp/src/hooks/core/use-withdraw-from-vault.tsx

```diff
@@ -40,6 +40,33 @@ export const useWithdrawFromVault = (vaultContractAddress: string, onSuccess?: C
     }
   };
 
+  // used in the modal
+  const withdrawFromVault = async (vaultAddress: string, amount: string) => {
```

Comment:
Because we don't have the vault address until we actually make the withdraw call. Whereas with `withdraw` we are using the known `activeVault`, we lack that when initializing the `useWithdrawFromVault` hook in the modal.